src/webapp/TourneyRent/src/app/pages/team-create/team-create/team-create.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-team-create',
+  templateUrl: './team-create.component.html',
+  styleUrls: ['./team-create.component.scss']
+})
+export class TeamCreateComponent {
+  Name!: string;
+  Description!: string;
+
+  constructor(private http: HttpClient) {}
+
+  createTeam() {
+    const team = {
+      Name: this.Name,
+      Description: this.Description
+    };
+
+    this.http.post('http://localhost:5155/Team', team).subscribe(response => {
+      console.log('Team created:', response);
```

Comment:
This could redirect somewhere where created team could be seen. Either in some list page where the teams will be displayed or something. You could check how I did it with tournament creation. I am not sure where the best place is to put that list. Maybe in profile page? Or if teams will be displayed for everyone, then you could do the same as I did with tournaments.

Reply:
Also would be great if you added a resource class e.g. like tournament.resource.ts, since I think later on when we do other things with teams we could reuse that logic in other components.